website/docs/concepts/html/events.mdx

```diff
@@ -128,7 +128,7 @@ use yew::prelude::*;
 
 #[function_component]
 fn MyComponent() -> Html {
-    let input_value_handle = use_state(String::default);
+    let input_value_handle = use_state(|| String::default);
```

Comment:
I believe the original usage is correct.

`use_state()` takes an init function which in this case is `String::default()`. Rust allows a shorthand syntax where a function can be passed with `String::default`.

Reply:
I've got 
use_state(String::default());
| --------- ^^^^^^^^^^^^^^^^^ expected an `FnOnce<()>` closure, found `String`

Reply:
You should use `use_state(String::default);` as demonstrated in the example. The extra bracket makes a function call which creates a string.